fapolicy_analyzer/ui/main_window.py

```diff
@@ -70,7 +73,10 @@ def __init__(self):
         self._fapdStopMenuItem = self.get_object("fapdStopMenu")
         self._fapd_status = ServiceStatus.UNKNOWN
         self._fapd_monitoring = False
+        self._fapd_ref = None
+        self._fapd_lock = Lock()
```

Comment:
I don't see where either of these are being used.

Reply:
Will remove; Artifacts of the original fapd monitoring implementation prior to moving it totally into fapd_manager.py

Reply:
Done. Will push shortly...